kong/pdk/tracing.lua

```diff
@@ -55,6 +55,8 @@ local SPAN_KIND = {
   CONSUMER = 5,
 }
 
+local EMPTY = {}
```

Comment:
here `EMPTY` is a module level var, will it pass dirty data from the previous request to the next request?

Reply:
It will never be modified, thus it's safe to share the same table.